docker/end-user-ui/Dockerfile

```diff
@@ -15,4 +15,4 @@ WORKDIR /tmp/end-user-ui/dist
 
 EXPOSE 8888
```

Comment:
Sorry for my ignorance of these things, but why not change this line as well?

Reply:
Good catch. Doesn't seem to matter, but still - clearly it's wrong, as is.